### PR TITLE
Initial Environment Setup

### PR DESCRIPTION
**Pull request by Shiara-Deudor**
-added styles (colors, defaults, and fonts)
-added fonts
-added custom server (.env)
-npm install dotenv
-npm install -D sass
-npm i -D @types/node

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
+    port: Number(process.env.PORT)
+  },
+  define: {
+    'process.env.PORT': `${process.env.PORT}`
+  }
 })
```
Comment (karl-jerico) [inline vite.config.js:16]: Maybe include the shortcut '@' for the file path.

**Review by mikco-hipe — APPROVED** [review]
LGTM

File: vite.config.js (hunk 1 above)
Comment (Shiara-Deudor) [reply vite.config.js:22]: I have resolved this 🙇

**Review by karl-jerico — APPROVED** [review]
LGTM

**Pull request merged**